addons/web/static/src/env.js

```diff
@@ -43,6 +43,13 @@ export function makeEnv() {
 
 const serviceRegistry = registry.category("services");
 
+serviceRegistry.addValidation({
+    start: Function,
+    dependencies: { type: Array, element: String, optional: true },
+    async: { type: Array, element: String, optional: true },
```

Comment:
fix declaration types

Reply:
Why don't we keep supporting true? It makes sense when the service exports a function, not an object

Reply:
@aab-odoo Which was present thrice in the code base:
In a test, in the bus service that didn't export a function (and crashed if instancied with useService) and in getColorPickerTemplateService which really shouldn't be a service.

If a service export a function, only the presence of the key is checked, so async: [] would work as well. 
Although it seemed a better idea the other day than now I have to say... 

An issue I have with async: true is that it's error prone for non veteran. Mainly because it should never have been called async in the first place, but more like asyncProtected or something.